sources/views/contacts.js

```diff
@@ -1,81 +1,89 @@
 import {JetView} from "webix-jet";
+import FormView from "views/form";
 import {contacts} from "models/contacts";
-import {textValidation} from "models/validate_function";
+
 export default class ContactsView extends JetView{
 	config(){
-		var contacts_list = {
+		const _ = this.app.getService("locale")._;
+
+		const add_new_button = {
+			view:"button",
+			value:_("Add new"),
+			css:"webix_primary",
+			click:()=>{
+				webix.message({text:_("New element was added"), expire:350});
+				const new_contact = { Name:"Name", Email:"Email", Status:0, Country:0};
+				contacts.add(new_contact);
+			}
+		};
+
+		const contacts_list = {
 			view:"list",
-			id:"contacts_list",
-			template:"#Name#, #Email#",
+			template:"#Name#, #Email# <span class='remove_list_item_btn webix_icon mdi mdi-close'></span>",
 			scrollY:true,
 			scrollX:false,
-			select:true
-		};
-		var contacts_form = {
-			view:"form",
-			id:"contacts_form",
-			width:350,
-			elements:[
-				{
-					margin:10,
-					rows:[
-						{template:"EDIT CONTACTS", type:"section"},
-						{view:"text", label:"Name", name:"Name", invalidMessage:"Enter correct Name"},
-						{view:"text", label:"Email", name:"Email", invalidMessage:"Enter correct Email"},
-						{margin:20, cols:[
-							{view:"button", value:"Save", css:"webix_primary", click:() => this.saveHandler("contacts_form")},
-							{view:"button", value:"Clear", css:"webix_secondary", click:() => this.clearHandler("contacts_form")}
-						]}
-					]
-				},{}
-			],
-			rules:{
-				Name:function(value){
-					return textValidation(value,20);
-				},
-				Email:webix.rules.isEmail
+			select:true,
+			on:{
+				onAfterSelect:function(id){
+					this.$scope.setParam("id", id, true);
+				}
+			},
+			onClick:{
+				remove_list_item_btn:function(e, id){
+					webix.confirm({
+						title:_("Contact data would be deleted"),
+						text:_("Do you still want to continue?"),
+						type:"confirm-warning"
+					}).then(() => {
+
+						webix.message({
+							text:_("Element was deleted"),
+							type:"info"
+						});
+
+						const prev_id = this.getPrevId(id);
```

Comment:
it is cheating. also - what if you remove the last item from the contacts?

how it should be solved: you show the url `top/contacts` and the contacts module decides in urlChange which item to select